ee/cis/win-10/cis-policy-queries-wip.yml

```diff
@@ -0,0 +1,283 @@
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Enabled: False (unchecked)':
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\BitLocker Drive Encryption\Operating System Drives\Choose how BitLocker-protected operating system drives can be recovered: Allow data recovery agent'
+    Note: This Group Policy path may not exist by default. It is provided by the Group Policy template VolumeEncryption.admx/adml that is included with the Microsoft Windows 7 & Server 2008 R2 Administrative Templates (or newer).
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\FVE\OSManageDRA' AND data = 0);
+  purpose: Informational
+  tags: compliance, CIS, CIS_LevelBL, CIS_win10_enterprise_1.12.0, CIS_bullet_18.9.11.2.4
+  contributors: rachelelysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Choose how BitLocker-protected operating system drives can be recovered: Recovery Password' is set to 'Enabled: Require 48-digit recovery password'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting allows you to control how BitLocker-protected operating system drives are recovered in the absence of the required startup key information. This policy setting is applied when you turn on BitLocker.
+    In "Configure user storage of BitLocker recovery information" select whether users are allowed, required, or not allowed to generate a 48-digit recovery password or a 256-bit recovery key.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to 'Enabled: Require 48-digit recovery password':
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\BitLocker Drive Encryption\Operating System Drives\Choose how BitLocker-protected operating system drives can be recovered: Recovery Password'
+    Note: This Group Policy path may not exist by default. It is provided by the Group Policy template VolumeEncryption.admx/adml that is included with the Microsoft Windows 7 & Server 2008 R2 Administrative Templates (or newer).
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\FVE\OSRecoveryPassword' AND data = 1);
+  purpose: Informational
+  tags: compliance, CIS, CIS_LevelBL, CIS_win10_enterprise_1.12.0, CIS_bullet_18.9.11.2.5
+  contributors: rachelelysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Choose how BitLocker-protected operating system drives can be recovered: Recovery Key' is set to 'Enabled: Do not allow 256-bit recovery key'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting allows you to control how BitLocker-protected operating system drives are recovered in the absence of the required startup key information. This policy setting is applied when you turn on BitLocker.
```

Comment:
is it ok for this one to have the same description as 18.9.11.2.5?

Reply:
Just looked them up, they have the same description and there's no extra notes to differentiate the description:

Screenshots:
<img width="687" alt="Screenshot 2023-04-20 at 3 32 44 PM" src="https://user-images.githubusercontent.com/71795832/233469538-1ad1c60a-e0e4-4884-9763-40c2cda2854f.png">
<img width="738" alt="Screenshot 2023-04-20 at 3 32 33 PM" src="https://user-images.githubusercontent.com/71795832/233469541-7e4736c9-f2e6-4fc3-8aaa-272d9c09943d.png">

@marcosd4h should we add something ourselves?

Reply:
thanks for looking at thi1